distribution/test/performance/jmeter/csw/pom.xml

```diff
@@ -49,14 +49,98 @@
                                         <target.port>${target.port}</target.port>
                                         <target.protocol>${target.protocol}</target.protocol>
                                     </propertiesJMeter>
-                                    <appendResultsTimestamp>true</appendResultsTimestamp>
-                                    <resultsFileNameDateFormat>YYYMMddHHmmss
-                                    </resultsFileNameDateFormat>
+                                    <testResultsTimestamp>false</testResultsTimestamp>
                                     <ignoreResultFailures>true</ignoreResultFailures>
                                 </configuration>
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>de.codecentric</groupId>
+                        <artifactId>jmeter-graph-maven-plugin</artifactId>
+                        <version>0.1.0</version>
+                        <executions>
+                            <execution>
+                                <id>create-graphs</id>
+                                <goals>
+                                    <goal>create-graph</goal>
+                                </goals>
+                                <phase>verify</phase>
+                                <configuration>
+                                    <inputFile>${project.build.directory}/jmeter/results/CSW-GetRecords.jtl</inputFile>
+                                    <graphs>
+                                        <graph>
+                                            <pluginType>ThreadsStateOverTime</pluginType>
+                                            <width>800</width>
```

Comment:
❓ I'm curious if there's a way to configure the width and height one time for all the graphs in this configuration block.

Reply:
let me look into this